backend/api/utils.py

```diff
@@ -1,15 +1,13 @@
 import json
-from bson import json_util, ObjectId
 from typing import Any
+from inflector import Inflector
```

Comment:
Nice tool, didn't heard about it.
<br>A Javascript version would be very helpful, ha ha #snakeToPascal

Reply:
Indeed. But in case you want to, pluralize is an npm package very useful for singularizing and pluralizing words ^^